x-pack/plugins/cross_cluster_replication/README.md

```diff
@@ -4,10 +4,11 @@
 
 You can run a local cluster and simulate a remote cluster within a single Kibana directory.
 
-1. Start your "local" cluster by running `yarn es snapshot --license=trial` and `yarn start` to start Kibana.
-2. Start your "remote" cluster by running `yarn es snapshot --license=trial -E cluster.name=europe -E transport.port=9400` in a separate terminal tab.
-3. Index a document into your remote cluster by running `curl -X PUT http://elastic:changeme@localhost:9201/my-leader-index --data '{"settings":{"number_of_shards":1,"soft_deletes.enabled":true}}' --header "Content-Type: application/json"`.
-Note that these settings are required for testing auto-follow pattern conflicts errors (see below).
+1. Ensure Kibana isn't running so it doesn't load up any data into your cluster. Run `yarn es snapshot --license=trial` to install a fresh snapshot. Wait for ES to finish setting up.
```

Comment:
@yuliacech I believe you wrote these originally, but I wasn't able to get the original instructions to work. Can you check this and see if I was just doing it wrong?

Reply:
Thanks for updating those instructions, @cjcenizal! I can confirm that the previous instructions didn't work for me either, tested your steps and it worked like a charm 👍